snips_helper/intent.py

```diff
@@ -24,9 +24,27 @@ def utterances(self):
         self.__deactivate()
         return training_data
 
+    def import_utterances(self, utterances):
+        print("Importing utterances")
+        self.__activate()
+        self.__get_utterance_dot_button().click()
+        self.__get_import_utterance_button().click()
+        self._get_modal_text_area().send_keys(utterances)
+        self.__get_modal_import_button().click()
+        time.sleep(1)
+        self.save()
+        return
+
+    def delete_utterances(self):
+        print("Importing utterances")
+        self.__activate()
+        while self.__get_utterance_delete_button():
+            self.__get_utterance_delete_button().click()
+        self.save()
+        return
```

Comment:
Same here, let's deactivate after the deletion is done that way all public methods are backed out to the bundle screen.